IndicoV2.V1Adapters/Jobs/JobStatusConverter.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using IndicoV2.Jobs.Models;
+
+namespace IndicoV2.V1Adapters.Jobs
+{
+    public class JobStatusConverter
+    {
+        public JobStatus Map(Indico.Types.JobStatus status) => (JobStatus)Enum.Parse(typeof(JobStatus), status.ToString());
```

Comment:
This does make sense for the interm moving from V1 -> V2 but if it hangs around any longer and someone inadvertently adds an enum in the wrong order it could cause an issue. Probably just something to add a quick test for in addition to the existing test

Reply:
[It's tested](https://github.com/IndicoDataSolutions/indico-client-csharp/blob/4cb83978abc091eb50143330f175d9e67b27a9d0/IndicoV2.V1Adapters.Tests/Jobs/JobStatusConverterTests.cs#L16)